RecoMuon/TrackingTools/python/MuonServiceProxy_cff.py

```diff
@@ -37,8 +37,10 @@
                                             'SmartPropagatorAnyRK',
                                             'StraightLinePropagator'),
         RPCLayers = cms.bool(True),
-        CSCLayers = cms.bool(True),
-        GEMLayers = cms.bool(False),
+        CSCLayers = cms.untracked.bool(True),
+        GEMLayers = cms.untracked.bool(True),
+        ME0Layers = cms.bool(True),
```

Comment:
this should be enabled somewhere else (2019 era or customizations), not in the default location